gremlin-go/README.md

```diff
@@ -72,9 +72,9 @@ Please review the [staticcheck documentation][scheck docs] for more details on i
 
 To install the Gremlin-Go as a dependency for your project, run the following in the root directory of your project that contains your `go.mod` file:
 
-`go get github.com/lyndonb-bq/tinkerpop/gremlin-go@gremlin-go-ms2`
+`go get github.com/lyndonb-bq/tinkerpop/gremlin-go@gremlin-go-ms5`
```

Comment:
If we are going to update this we should update it to be apache/tinkerpop/gremlin-go@3.5-dev